test/services/incoming_notification_service_test.rb

```diff
@@ -36,7 +36,7 @@ class LockingTest < ActiveSupport::TestCase
     self.use_transactional_tests = false
 
     def setup
-      skip "see https://github.com/rails/rails/pull/46553" if Rails.version <= Gem::Version.new('7.0.6')
+      skip 'see https://github.com/rails/rails/pull/46553' if Rails.version <= Gem::Version.new('7.0.6')
```

Comment:
why use single quotes? Not a problem, just asking. I thought double quotes are generally preferred always unless they enclose other double quotes.

Reply:
I think both are equally correct, as long as they are consistent: https://github.com/rubocop/ruby-style-guide#consistent-string-literals

I think this project mostly uses "default single quote" style, where double quote is only used when interpolation is needed. I personally also prefer this one.